docs/tools/piral-inspector.md

```diff
@@ -4,4 +4,203 @@ title: Piral Inspector
 
 # Piral Browser Extension
 
-(tbd)
+## Introduction
+
+To optimally work as a developer with your Piral Instance, and all of its containing Pilets, Piral Inspector offers a great set of capabilities:
+- Getting information about the currently running Piral Instance, e.g. name and version
+- Overview on currently loaded Pilets
+- Adding / Removing of Pilets on-the-fly
+- Overview of registered routes
+- Collection of emitted events
+- Collection of global states
+
+As a developer of your Piral Instance, or your Pilets, you are used to working with a web browser. And this is precisely the best place to get assisted by the Piral Inspector during the development or testing phase.
+
+We provide the Piral Inspector as an extension for the following browsers:
+- Google Chrome (>= v50)
+- Firefox (>= v50)
+- Opera (>= v50)
+
+## Installation of Add-On / Extension
+
+::: summary: Firefox
+
+Type the following URL into the browser address line followed by hitting enter:
+
+about:addons
+
+On the following page, you can search for "Piral Inspector", and you will see the add-on as the first result:
+
+![FirefoxAddOnSearchResultPiralInspector](../diagrams/piralinspector_firefox_add-on_searchresult.png)
+
+Click on "Piral Inspector".
+
+On the following details page, click on "Add to Firefox":
+ 
+![FirefoxAddOnAddToFirefoxPiralInspector](../diagrams/piralinspector_firefox_add-on_add.png)
+
+Now, a security message will appear, please click on "Add" to accept the required security settings:
+
+![FirefoxAddOnSecurityMessagePiralInspector](../diagrams/piralinspector_firefox_add-on_security.png)
+
+Piral Inspector is now available, and you can open it via the Web Developer Tools (```Tools > Web Developer > Toggle Tools```).
```

Comment:
Analog den anderen changes -> bitte backticks entfernen.

Reply:
Got it.